src/components/todo/index.js

```diff
@@ -56,26 +71,21 @@ class TodoPage extends Component {
             placeholder="Enter a new item here. Then press ENTER"
             value={this.state.newItem} // display @observable newItem in this textfield
             onChange={(event) => {
-              // TODO: record the change when user update the textfield
               this.setState({ newItem: event.target.value })
-              console.log('this.state.newItem', this.state.newItem)
             }}
             onKeyPress={(event) => { // when a keyboard is pressed
-              // when 'ENTER' is pressed
               if (event.key === 'Enter' && this.state.newItem) {
-                // TODO: add the item user has enter into the list
                 this.setState({
                   todoItems: [
                     ...this.state.todoItems,
                     {
-                      id: this.state.newItem,
+                      id: this.state.todoItems.length + 1,
```

Comment:
normally react will say dont use the array length as the `key` 
because if id = 1, and if you remove the item, a new id = 1 will point to a different object in the array but the html will not change because key is still 1
`<div key={item.id} className={s.todoItem}>` id is used here as key